packages/ra-core/src/controller/useEditController.spec.tsx

```diff
@@ -69,7 +58,7 @@ describe('useEditController', () => {
             </EditController>
         );
         act(() => saveCallback({ foo: 'bar' }));
-        const crudUpdateAction = dispatch.mock.calls[2][0];
+        const crudUpdateAction = dispatch.mock.calls[1][0];
```

Comment:
instead of relying on the dispatch mock call index, I tend to do a `find` using the action type to make the test future proof. Not necessary for this PR though.